data/rescue/rescue.file_list

```diff
@@ -378,6 +378,7 @@ R s/ Server//g etc/issue
 # remove files we don't want to show up at all
 r /usr/share/{doc,info,locale,man} /usr/src/packages /usr/share/fillup-templates /var/adm/fillup-templates
 r dev/mapper dev/stderr dev/initctl
+e find . -name __pycache__ -print0 | xargs -0 rm -rf
```

Comment:
I do not know where the cached files can be located, but do we need to scan the whole tree? How long does it take in OBS? I mean scanning e.g. just `usr` could be a bit faster... But maybe it's irrelevant considering how long the i-i build actually takes....

Reply:
I don't think it would make much of a difference. `/usr` is the largest blob anyway and the file tree has just been created
and is likely still cached.

Reply:
Luckily the measurement is easily obtained, and the answer is: it takes 1 second or less:
```console
$ grep -A1 pycache /var/tmp/build-root-openSUSE_Factory/.build.log
[  333s] running "find . -name __pycache__ -print0| xargs -0 rm -rf"
[  334s] running "touch etc/sysconfig/kernel"
--
[  475s] running "find . -name __pycache__ -print0| xargs -0 rm -rf"
[  475s] running "find var -type f"
--
[ 1348s] running "find . -name __pycache__ -print0| xargs -0 rm -rf"
[ 1348s] running "touch etc/sysconfig/kernel"
```